runtime/runc_jailer.go

```diff
@@ -294,13 +294,13 @@ func (j *runcJailer) BuildLinkFifoHandler() firecracker.Handler {
 	return firecracker.Handler{
 		Name: jailerFifoHandlerName,
 		Fn: func(ctx context.Context, m *firecracker.Machine) error {
-			logFifo, err := j.makeLinkInJail(m.Cfg.LogPath)
+			logFifo, err := j.makeLinkInJail(m.Cfg.LogPath, internal.FirecrackerLogFifoName)
```

Comment:
Does this mean `LogPath` will always be a directory and not a path to a file? Cause if that is the intention I dont know if I like the change.

Reply:
No. LogPath is still the path to Firecracker's log file. Previously the basename part of LogPath is used, but it restricts the value LogPath can take. For example, using `/var/log/fc/log/[VM ID].log` as LogPath while using `/var/log/fc/metrics/[VM ID].log` doesn't work since both of them become `[VM ID].log` inside a jail directory.